src/test/java/org/kainos/ea/integration/IntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+package org.kainos.ea.integration;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.DropwizardApplicationApplication;
+import org.kainos.ea.DropwizardApplicationConfiguration;
+import org.kainos.ea.cli.Login;
+import org.kainos.ea.cli.Register;
+import io.dropwizard.configuration.ResourceConfigurationSourceProvider;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.kainos.ea.cli.Role.Admin;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class IntegrationTest {
+    static final DropwizardAppExtension<DropwizardApplicationConfiguration> APP = new DropwizardAppExtension<>(
+            DropwizardApplicationApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+    @Test
+    void postRegister_shouldReturn200OnSuccess() {
+        Register register = new Register("test@example.com", "password123", Admin);
```

Comment:
I'm assuming this works because these are valid login details which we wouldn't want to store in source, please pass these in via ENV VARs

Reply:
@shaunganley I was running this test and it Works ok for new data but once there is duplicate email it will fail as it checks for duplicate email so the will I put in the ENV VAR and just change it every time it runs?